core/domain/opportunity_services.py

```diff
@@ -498,27 +463,50 @@ def get_translation_opportunities(language_code, topic_name, cursor):
         .ExplorationOpportunitySummaryModel.get_all_translation_opportunities(
             page_size, cursor, language_code, topic_name))
     opportunities = []
-    suggestion_ids = []
-    opportunity_ids = []
+    exp_id_to_in_review_count = (
+        _build_exp_id_to_translation_suggestion_in_review_count(
+            exp_opportunity_summary_models, language_code))
+    for exp_opportunity_summary_model in exp_opportunity_summary_models:
+        exp_opportunity_summary = (
+            get_exploration_opportunity_summary_from_model(
+                exp_opportunity_summary_model))
+        if exp_opportunity_summary.id in exp_id_to_in_review_count:
+            exp_opportunity_summary.translation_in_review_counts = {
+                language_code: exp_id_to_in_review_count[
+                    exp_opportunity_summary.id]
+            }
+        opportunities.append(exp_opportunity_summary)
```

Comment:
This reads a bit weirdly -- what is the difference between an "opportunity" and an "opportunity summary"? (I guess this was an issue in the previous version of the code, too, but we should probably standardize the naming.)

Reply:
Aha, I suspect that maybe it was named like that to distinguish it from "opportunity" because of the extra field. That way, "opportunity" refers to stuff without the field and "opportunity summary" refers to stuff with the field.

Is that consistent with what's actually happening in the codebase? If so, perhaps worth fixing the naming/comments to clarify that (or doing a larger refactor as suggested above).